### PR TITLE
Davias97 username/password validation

### PR DESCRIPTION
**Pull request by Davias97**

### CODE REVIEW

File: src/main/java/com/example/todoz/controllers/AuthenticationController.java [hunk 1]
```diff
@@ -47,14 +47,26 @@ public String register(RegisterDTO registerDTO, RedirectAttributes ra) {
...
+        } else {
+            ra.addFlashAttribute("usernameWrong", true);
+            return "redirect:/register";
         }
 
         return "redirect:/login";
     }
+
+    public boolean validateUsername(String username) {
+        return !username.isBlank() && username.length() >= 3;
```
Comment (dovantuan97) [inline src/main/java/com/example/todoz/controllers/AuthenticationController.java:66]: password should not generally contain " " and this method doesn't check it

File: src/main/java/com/example/todoz/controllers/AuthenticationController.java [hunk 2]
```diff
@@ -47,14 +47,26 @@ public String register(RegisterDTO registerDTO, RedirectAttributes ra) {
...
+    }
+
+    public boolean validatePassword(String password) {
+        return !password.isBlank() && password.length() >= 5;
```
Comment (dovantuan97) [inline src/main/java/com/example/todoz/controllers/AuthenticationController.java:70]: same issue

File: src/main/java/com/example/todoz/controllers/AuthenticationController.java (hunk 1 above)
Comment (Davias97) [reply src/main/java/com/example/todoz/controllers/AuthenticationController.java]: whitespace validation added)

File: src/main/java/com/example/todoz/controllers/AuthenticationController.java (hunk 2 above)
Comment (Davias97) [reply src/main/java/com/example/todoz/controllers/AuthenticationController.java]: fixed

**Pull request closed without merging**